### PR TITLE
task: setup basic agones sdk actions and introduce autoscaler

### PR DESCRIPTION
**Pull request by syllabix**

### CODE REVIEW

File: server/src/main.rs [hunk 1]
```diff
@@ -2,29 +2,52 @@ mod board;
...
 use actix_cors::Cors;
 
 use actix_web::middleware::Logger;
 use actix_web::{web, App, HttpServer};
 
-use crate::board::Registry;
+use crate::board::{HealthChecker, Registry};
 
-#[actix_web::main]
+#[tokio::main]
```
Comment (muratozkan) [inline server/src/main.rs:15]: do we need to boot an actor system manually in this case? or does the `HttpServer` do that for us?

File: agones/helm/values.yaml [hunk 2]
```diff
@@ -1,4 +1,35 @@
 # Overrides values from: https://github.com/googleforgames/agones/blob/release-1.26.0/install/helm/agones/values.yaml
+agones:
```
Comment (muratozkan) [inline agones/helm/values.yaml:2]: maybe we can tag values that are different than the defaults?

File: server/Cargo.toml [hunk 3]
```diff
@@ -21,12 +21,8 @@ random_color = "0.6.1"
...
 serde_json = "1.0.85"
 serde_millis = "0.1.1"
-tokio = { version = "1.21.2", features = ["macros", "sync"] }
-
-[dependencies.agones]
-git = 'https://github.com/googleforgames/agones.git'
-branch = 'release-1.26.0'
-optional = true
+tokio = { version = "1.21.2", features = ["full"] }
+agones = { git = 'https://github.com/googleforgames/agones.git', branch = 'release-1.26.0', optional = true }
```
Comment (muratozkan) [inline server/Cargo.toml:25]: :rocket:

**Review by muratozkan — APPROVED** [review]
just minor comments for my information mostly - looks great otherwise :+1:

File: server/src/main.rs (hunk 1 above)
Comment (syllabix) [reply server/src/main.rs:15]: yep - looks like actix web boots up the actors.

File: agones/helm/values.yaml (hunk 2 above)
Comment (syllabix) [reply agones/helm/values.yaml:2]: agreed - I will add comments here just so this is clear (and to remind our future selves why this is being done this way 😅 )

**Comment by syllabix** [comment]
@muratozkan - thanks for the review! I was able to use cargo features to implement a game server manager without the need of a trait!

**Pull request merged**